homeassistant/loader.py

```diff
@@ -78,6 +78,7 @@
     "repairs",
     "system_health",
     "trigger",
+    "backup",
```

Comment:
Its too late to add via `async_register_preload_platform` because `hassio`, `cloud`, and `recorder` are loaded very early